go.mod

```diff
@@ -86,7 +86,7 @@ require (
 	k8s.io/cri-api v0.27.1
 	k8s.io/klog/v2 v2.90.1
 	k8s.io/utils v0.0.0-20230220204549-a5ecb0141aa5
-	tags.cncf.io/container-device-interface v0.7.2
+	tags.cncf.io/container-device-interface v0.8.1
```

Comment:
Shouldnt this be first updated in release/2.0 https://github.com/containerd/containerd/blob/release/2.0/go.mod#L88 and then make the change here. Otherwise we will have an older version in the latest 2.0 branch

Reply:
Yes. I probably filed the PRs (this and #11430 in the wrong order.

Reply:
Could you open a PR against `main` as well? Otherwise we may regress if a new release is created from `main` and it doesn't have this update.

Reply:
The usual flow is for updates to go into main, then back ported (or equivalent) into the release branches.